code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -456,6 +534,8 @@
 ///Nulls the target of our hunter's mark
 /datum/action/ability/activable/xeno/hunter_mark/proc/unset_target()
 	SIGNAL_HANDLER
+	to_chat(owner, span_xenodanger("We cannot maintain our focus on [marked_target] any longer."))
```

Comment:
Pretty sure Tivi meant REPLACE IT with a balloon alert, not have both. I don't care though

Reply:
I think this way it's better since it puts a log in the chat aswell, I thought. No?